lib/presentation/book/bloc/home_page_bloc.dart

```diff
@@ -1,5 +1,5 @@
 import 'package:book/data/firebase_firestore/firebase_db_manager.dart';
-import 'package:book/models/book.dart';
+import 'package:book/models/book/book.dart';
 import 'package:book/presentation/book/bloc/home_page_event.dart';
```

Comment:
instead of importing files for events and state, you could use "part" and "part of".